tests/cryptolibAESSHA/cryptolibAESSHA.ino

```diff
@@ -4,7 +4,9 @@ Uses 512 Byte block examples
 */
 #include "CryptoAccel.h" //Makes it arduino compatible
 #include "cau_api.h"
+
 #define data_size 512
+#define AES_ROUNDS 10
```

Comment:
not quite descriptive enough; This is a *value* for the number of AES rounds, true; however, for AES other than AES-128 it is not a valid number of rounds...

Reply:
Easy change. I replaced the one @dcn1470 entered too.